nebula.proto

```diff
@@ -23,13 +23,18 @@ message NebulaMeta {
 }
 
 message NebulaMetaDetails {
-  uint32 VpnIp = 1;
+  Ip VpnIp = 1;
   repeated Ip4AndPort Ip4AndPorts = 2;
   repeated Ip6AndPort Ip6AndPorts = 4;
-  repeated uint32 RelayVpnIp = 5;
+  repeated Ip RelayVpnIp = 5;
   uint32 counter = 3;
 }
 
+message Ip {
```

Comment:
same comment as above, "what if we used `bytes`?"

Reply:
I dislike the unbounded nature of `bytes`. 

As mentioned in the other comment, I had noodled on something along these lines
```
message NebulaMetaDetails {
  fixed64 VpnIpHi = 1;
  uint64 VpnIpLo = 2;
  ...
}
```

Stuff to noodle on at least